Source/WebCore/platform/image-decoders/bmp/BMPImageReader.cpp

```diff
@@ -392,20 +394,23 @@ bool BMPImageReader::processBitmasks()
         // we read the info header.
 
         // Fail if we don't have enough file space for the bitmasks.
+        const size_t headerEnd = m_headerOffset + m_infoHeader.biSize;
+        const size_t bitmasksSize = 12;
+        const size_t bitmasksEnd = headerEnd + bitmasksSize;
         static const size_t SIZEOF_BITMASKS = 12;
```

Comment:
This definition can be removed now

Reply:
Forgot.. Good catch!

Reply:
Fixed in [a867c98](https://github.com/WebKit/WebKit/commit/a867c98a3ae1b3a76497fcd68dab5391896454dc)